### DISCUSSION TITLE
Join closed-testing release channel

### DISCUSSION THREAD

**Discussion by menny** [AnySoftKeyboard/AnySoftKeyboard#3391] [General]
The _alpha_ channel for the keyboard and the language packs is a closed-testing group, meaning you'll need to be added manually to the group.
Alpha distribution is done using Google's Play Store, meaning you will need to install AnysFotKeyboard and Language packs using Play Store.

_AnySoftKeyboard app:_
For the app, join the group: https://groups.google.com/g/anysoftkeyboard-alpha-testers and then approve joining the testing alpha here: https://play.google.com/apps/testing/com.menny.android.anysoftkeyboard

_Language packs:_
Ask to be added to this discussion.

**Comment by mhdzli** [comment]
Hi
Whould you please add me on alpha channel group for testing the Persian language pack.
My Gmail account is mz7512@gmail.com
Thanks

**Reply by menny** [reply]
Done. Should be available for you in the Play Store in a couple of hours

**Comment by mhdzli** [comment]
Thank you

**Comment by nihal697** [comment]
Hi Menny,
Id also like to join the alpha channel. I will send you an email containing my mail id. I do not want to put it here for privacy reasons, hope you understand.
Thanks

**Reply by menny** [reply]
For specific language? Or the AnySoftKeyboard main keyboard app?
For the app, join the group: https://groups.google.com/g/anysoftkeyboard-alpha-testers and then approve joining the testing alpha here: https://play.google.com/apps/testing/com.menny.android.anysoftkeyboard

**Comment by nihal697** [comment]
for main app.

**Comment by nihal697** [comment]
I am already in them, but the last update I got from play store was
v1.11.582 in august.

**Reply by menny** [reply]
Currently, the last update on that channel is `Release: 1.11.660 Last updated: Oct 26, 2022` Are you using Play Store?

**Comment by nihal697** [comment]
yes i am using playstore.
i installed the same version yesterday from github actions.

**Comment by nihal697** [comment]
isnt it possible to push alpha builds from some other way besides play
store ? i have about 20 frnds who do not use play store but love ASK. so I
have to give them the apk with each update.( I hope that's okay with you).

**Comment by luciobortoletto** [comment]
Hi,
Would you please add me on alpha channel group for testing the Italian language pack.
My Gmail account is lucio.bortoletto.lb@gmail.com
Thanks

**Comment by riddle-rhymer** [comment]
Play Store says it's for an older version of Android and so won't let me download.

I have Android 13.
